tests/core/math/test_color.h

```diff
@@ -144,6 +144,24 @@ TEST_CASE("[Color] Conversion methods") {
 			"The string representation should match the expected value.");
 }
 
+TEST_CASE("[Color] Linear <-> sRGB conversion") {
+	const Color color = Color(0.35, 0.5, 0.6, 0.7);
+	const Color color_linear = color.to_linear();
+	const Color color_srgb = color.to_srgb();
+	CHECK_MESSAGE(
+			color_linear.is_equal_approx(Color(0.1005, 0.214, 0.3185, 0.7)),
```

Comment:
This test fails and I have no idea why – it's matching the exact value when `color_linear` is printed. Same for `color_srgb` below.

The bottom two tests are successful though.

Reply:
I wonder if this something we should fix in the sRGB/linear conversion functions, or just use a custom epsilon for this check.

Reply:
Is it actually losing precision? Seems to me like the issue is that you are using a reference value which has been rounded to 0.003 and that's lossy/only for pretty printing.

Reply:
Is it loosing precision? I'm getting the same value with the calculator, e.g., `R = 0,1004815099`. I guess it's `print` what's loosing precision.

Reply:
Indeed, the print was only printing 4 decimals (both in GDScript and C++). I'm not sure if it should be rounding this much.